tooling/nargo_cli/tests/compile_failure/dup_trait_implementation/src/main.nr

```diff
@@ -1,26 +0,0 @@
-use dep::std;
-
-trait Default {
-    fn default(x: Field, y: Field) -> Field;
-}
-
-struct Foo {
```

Comment:
Why were all these integration tests removed? Do we have equivalents for them already or was it an accident?

Reply:
They are transformed into unit tests. 

As an integration test, we can only check, if the codes fail to compile for any reason. This approach is prone to allow regression of the code quality. Image, we accidentally introduce a bug that produce an extra error. This will probably not be noticed in the integration test. 


As an unit test, I check for the exact reason why this sample code does not compile. This is very useful to avoid shadowing new defects in the code as we develop noir in the future.